causal_testing/testing/estimators.py

```diff
@@ -134,6 +150,60 @@ def estimate_ate(self) -> (float, [float, float], float):
         confidence_intervals = list(t_test_results.conf_int().flatten())
         return ate, confidence_intervals
 
+    def estimate_risk_ratio(self) -> (float, [float, float], float):
```

Comment:
maybe use
def estimate_risk_ratio(self) -> Tuple[float, Sequence[float, float], float

(If I add this in a code block it crashes?)